README.md

```diff
@@ -1,119 +1,51 @@
-# Kitkit School
+## Pehla School
 
-Kitkit School (phase 3) consists of 13 apps as following:
+Pehla School consists of 2 apps as following:
 
-1. Launcher
-* A home screen app, from where other apps are launched.
-
-2. Lock Screen
-* This app appears when the device wakes up.
-* The device will be unlocked by touching the Kitkit School icon in the middle.
+1. Pehla School
+* Main learning app with tailored curriculum.
 
-3. Library
+2. Pehla School Library
 * A collection of videos and books for children to enjoy.
 * Video will play in place, while a separate book viewer will open to read a book.
```

Comment:
Specify what 'in place' here means like same window.

Reply:
This line is not required anymore